content/blog/2018/08/2018-08-14-code-coverage-api-plugin-1.0-release.adoc

```diff
@@ -0,0 +1,205 @@
+
+=== Supported Tools
+
+* Embedded tools
+** JaCoCo
+* Other plugins as an Extension of Code Coverage API plugin
+** Cobertura (https://github.com/jenkinsci/cobertura-plugin[Cobertura Plugin])
+** llvm-cov (https://github.com/jenkinsci/llvm-cov-plugin[llvm-cov Plugin])
+
+=== Features
+
+* Modernized coverage chart
+* Coverage trend
+* Source code navigation
+* Parallel pipeline support
+* Reports combining
+* Rest API
+* Failed conditions and flexible threshold setting
+* Other small features
+
+==== Modernized coverage chart
+
+In the summary chart we can see the coverage summary of current coverage metric.
+image:/images/post-images/2018-06-13-code-coverage-api-plugin/summary-chart.gif[title="Summary Chart", role="center"]
+
+==== Child Summary Chart
+
+In the Child summary chart, we can see the coverage summary of each child, also, we can use the range handler to filter item we want to see to reduce the chart size. If we want see coverage details of child, we can click the child name to see more information.
+image:/images/post-images/2018-06-13-code-coverage-api-plugin/child-summary-chart.gif[title="Child Summary Chart", role="center"]
+
+=== Coverage Trend
+
+We also support coverage trend to show coverage metrics changing between builds.
+image:/images/post-images/2018-06-13-code-coverage-api-plugin/trend-chart.gif[title="Trend Chart", role="center"]
+
+==== Source code navigation
+You can enable source code navigation by specifying Source File Storing Level to save last build source files (enable source files navigation in current and last build) or save all build source files (enable source files navigation in all builds).
+image:/images/post-images/code-coverage-api/source-files-config.png[title="Source File Navigation Configuration" role="center"]
+
+You can see source file with coverage information on File level coverage page.
+image:/images/post-images/code-coverage-api/source-files-result.png[title="Source File Navigation Result" role="center"]
+
+==== Parallel pipeline support
+
+We support parallel pipeline. You can call Code Coverage API plugin in different branches like that:
+[source, groovy]
+----
+node {
+    parallel firstBranch: {
+        publishCoverage adapters: [jacoco('target/site/jacoco/jacoco.xml')]    
+}, secondBranch: {
+        publishCoverage adapters: [jacoco('jacoco.xml')]
+    }
+}
+----
+
+==== Reports combining
+You can add tag on `publishCoverage` and Code Coverage API plugin will combine reports have same tag
+
+----
+node {
+    parallel firstBranch: {
+        publishCoverage adapters: [jacoco('target/site/jacoco/jacoco.xml')], tag: ‘t’
+}, secondBranch: {
+        publishCoverage adapters: [jacoco('jacoco.xml')], tag: ‘t’
+    }
+}
+----
+
+==== Rest API
+We provide REST API for others to retrieve coverage data:
```

Comment:
`a REST API`
Also, I suggest removing `for others` - it's clear to me who the others are, and probably not worth elaborating